### PR TITLE
Create User class and Project class

### PR DESCRIPTION
**Pull request by chelsea-chenxi1999**

### CODE REVIEW

File: backend/src/main/java/com/itlize/joolemarketplace/model/Project.java
Comment (tak-oshima) [inline backend/src/main/java/com/itlize/joolemarketplace/model/Project.java:1]: Can we add a `@OneToMany` relationship with `ProjectProduct`?

File: backend/src/main/java/com/itlize/joolemarketplace/model/User.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.LazyCollection;
+import org.hibernate.annotations.LazyCollectionOption;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+import org.springframework.stereotype.Component;
```
Comment (tak-oshima) [inline backend/src/main/java/com/itlize/joolemarketplace/model/User.java:11]: Lets clean up unused imports here

File: backend/src/main/java/com/itlize/joolemarketplace/model/Project.java
Comment (ZhangShuoyu) [reply backend/src/main/java/com/itlize/joolemarketplace/model/Project.java:1]: Yeah, I think we need a @OneToMany here.

Comment (ZhangShuoyu) [reply backend/src/main/java/com/itlize/joolemarketplace/model/Project.java:1]: And also the corresponding getter and setter?

Comment (chelsea-chenxi1999) [reply backend/src/main/java/com/itlize/joolemarketplace/model/Project.java:1]: Okay!

File: backend/src/main/java/com/itlize/joolemarketplace/model/User.java (hunk 1 above)
Comment (chelsea-chenxi1999) [reply backend/src/main/java/com/itlize/joolemarketplace/model/User.java:11]: Okay! Finished that and I'll commit again.

**Pull request closed without merging**